messaging/messaging/src/main/java/io/helidon/messaging/MessagingException.java

```diff
@@ -30,4 +30,14 @@
     public MessagingException(final String message) {
         super(message);
     }
+
+    /**
+     * Create new MessagingException with supplied message and cause.
+     *
+     * @param message supplied message
+     * @param cause   of this exception
+     */
+    public MessagingException(final String message, Throwable cause) {
```

Comment:
we do not use `final` in parameter declaration

Reply:
fixed